import java.util.Scanner;

public class exceptionhandling {

    public float  informationGathering() throws Exception{
        int a=0; int b = 23;
        float result = b/a;
        return result;
    }
    public String nameOfStudent() throws ArithmeticException{
        System.out.println("Name : ");
        Scanner sc = new Scanner(System.in);
        String name = sc.nextLine();
        System.out.println(name.toUpperCase());

        return name;
    }
    public static void main(String[] args) {
        exceptionhandling one = new exceptionhandling();
        try{
            one.nameOfStudent();
            one.informationGathering();
        }catch(Exception ex){
            System.out.println(ex.getMessage());
        }

    }
}
